Logic/item_detection.py

```diff
@@ -0,0 +1,196 @@
+'''Item Detection Module'''
+
+import os
+import numpy as np
+import cv2
+
+from skimage.measure import compare_ssim as ssim
+
+from Config.eight_ball_lookup import EightBallLookup
+
+class ItemDetection:
+    '''Responsible for detecting game balls and holes'''
+
+    lookup = EightBallLookup()
+
+    def identify_parameters(self, for_holes, for_balls):
```

Comment:
Add comment to clarify parameter type and use

Reply:
Done